build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
+
 buildDir="build"
-rm -rf "$buildDir"
-mkdir "$buildDir"
+mkdir -p "$buildDir"
```

Comment:
Why remove the `rm`? The build directory should be cleared to make sure there aren't files from previous builds, no?

Reply:
I see no reason to delete previously builds. It is in the .gitignore file anyways and the build files are named with their version number.

Reply:
It's just pretty standard practice for a build script. 1) No reason to keep accumulating files in the build directory, and 2) if you didn't like the build, reverted the commits, changed something, and built again, you could end up with an XPI that contained files you weren't expecting. So it has to at least delete the old XPI, but I don't think there's any reason not to just clear the directory.

Reply:
(If you build the same version multiple times then the new build overwrites the older one.)

But I can simply change it back, if this is standard practice. No strong reasons against it.

Reply:
> If you build the same version multiple times then the new build overwrites the older one.

`zip` doesn't overwrite an existing file if one exists — it updates it. So without an `rm`, the ZIP file could contain files that were deleted. Unlikely to be an issue, since anything referencing them would likely be overwritten, but no reason not to start fresh.